climpred/bootstrap.py

```diff
@@ -24,24 +24,24 @@
 )
 
 
-def _resample(hind, shuffle_dim, to_be_shuffled):
-    """Resample with replacement in dimension `shuffle_dim` from values of
-    `to_be_shuffled`
+def _resample(hind, resample_dim, to_be_resampled):
+    """Resample with replacement in dimension `resample_dim` from values of
+    `to_be_resampled`
 
     Args:
         hind (xr.object): input xr.object to be shuffled.
-        shuffle_dim (str): dimension to shuffle along.
-        to_be_shuffled (list, xr.DataArray.values, np.ndarray): values to shuffle from.
+        resample_dim (str): dimension to shuffle along.
+        to_be_resampled (list, xr.DataArray.values, np.ndarray): values to shuffle from.
 
     Returns:
-        xr.object: shuffled along `shuffle_dim`.
+        xr.object: shuffled along `resample_dim`.
 
     """
-    smp = np.random.choice(to_be_shuffled, len(to_be_shuffled))
-    smp_hind = hind.sel({shuffle_dim: smp})
+    smp = np.random.choice(to_be_resampled, len(to_be_resampled))
+    smp_hind = hind.sel({resample_dim: smp})
     # ignore because then inits should keep their labels
-    if shuffle_dim != 'init':
-        smp_hind[shuffle_dim] = np.arange(1, 1 + smp_hind[shuffle_dim].size)
+    if resample_dim != 'init':
+        smp_hind[resample_dim] = np.arange(1, 1 + smp_hind[resample_dim].size)
```

Comment:
Should this be `smp_hind[resample_dim] = hind[resample_dim]` in case there's unique labeling to the members, for instance?

Reply:
your suggestion is more straightforward. yes